src/allotropy/parsers/utils/values.py

```diff
@@ -196,3 +197,16 @@ def num_to_chars(n: int) -> str:
 
 def str_or_none(value: Any) -> Optional[str]:
     return None if value is None else str(value)
+
+
+def get_val_from_xml(
+    xml_object: ElementTree.Element,
+    tag_name: str,
+) -> str:
+    val_from_xml = xml_object.find(".//" + tag_name)
```

Comment:
I think it would be best to require a full path (as you note in your comment to me)

This reduces chance of an error occurring because there is a different, unexpected tag with the same name elsewhere in the document.